src/Serilog/Configuration/LoggerSinkConfiguration.cs

```diff
@@ -128,9 +128,21 @@ internal LoggerSinkConfiguration(LoggerConfiguration loggerConfiguration, Action
             var lc = new LoggerConfiguration();
 
             _applyInheritedConfiguration(lc);
-
             configureLogger(lc);
-            return Sink(new SecondaryLoggerSink(lc.CreateLogger(), attemptDispose: true), restrictedToMinimumLevel, levelSwitch);
+
+            var subLogger = lc.CreateLogger();
+            if (subLogger.HasOverrideMap)
+            {
+                // Overrides in sub-loggers are not supported and will be ignored
+                SelfLog.WriteLine("We have detected that you are using .MinimumLevel.Override() from a sub-logger (.WriteTo.Logger(Action<LoggerConfiguration>)). " +
+                                  "Overrides in a sub-logger are not supported and are therefore ignored. " +
+                                  "We recommend applying .MinimulLevel.Override on the root logger or " +
+                                  "using .Filter in your sub-loggers. " +
+                                  "Note that support for sub-loggers' minimum level overrides may be removed in the next major version.");
```

Comment:
Reads very nicely 👍

We could probably cut it down a bit to just:

`Minimum level overrides are not supported on sub-loggers and may be removed completely in a future version.`

since it's googleable back to this issue :-) Happy to go with the longer version if you think it's preferable.